app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -21,14 +28,42 @@ import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp52
 import co.nimblehq.compose.crypto.ui.theme.Style
 import co.nimblehq.compose.crypto.ui.theme.Style.textColor
 import co.nimblehq.compose.crypto.ui.uimodel.CoinItemUiModel
+import co.nimblehq.compose.crypto.ui.userReadableMessage
 
 @Suppress("FunctionNaming", "LongMethod")
 @Composable
 fun HomeScreen(
     viewModel: HomeViewModel = viewModel()
 ) {
+    val context = LocalContext.current
+    LaunchedEffect(Unit) {
```

Comment:
Are you referring from [this answer](https://stackoverflow.com/a/71374631) to handle showing Toast case?

Reply:
@hoangnguyen92dn Tested, it's still working fine. I'll leave it for later check if we need to do something more.


https://user-images.githubusercontent.com/16315358/189077940-8292fe83-e658-4a49-b21e-21d5f06843fd.mp4